contents/blog/the-posthog-array-1-43-0.md

```diff
@@ -21,16 +21,16 @@ Any notes about async migrations, etc. go here.
 **Release highlights:**
 
 - [New: Performance improvements](#new-performance-improvements)
```

Comment:
Also newwww incoming!: https://github.com/PostHog/posthog/pull/13611/files

Permissioning on feature flags, for enterprise

Reply:
I've also created a manual docs for this, can someone take a look at it? :) https://github.com/PostHog/posthog.com/pull/5008